UnhollowerBaseLib/ClassInjector.cs

```diff
@@ -20,6 +20,8 @@ namespace UnhollowerRuntimeLib
 {
     public unsafe static class ClassInjector
     {
+        public delegate Type[] FindImplementsDelegate(Type type);
```

Comment:
Why not just use `Func<Type, Type[]>`?

Reply:
It wouldn't match the style of the other methods.

Reply:
What other methods? I only see native delegates here that have to be made like that